src/components/WalletModal/WalletModal.tsx

```diff
@@ -75,157 +78,208 @@ const WalletModal: React.FC<ModalProps> = ({ isOpen, onDismiss }) => {
+          </Box>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'Staked Uniswap ETH/DPI LP Icon',
+                src: 'https://set-core.s3.amazonaws.com/img/coin-icons/uni_lp.svg',
+              }}
+              label='Staked Uniswap ETH/DPI LP'
+              link={`https://etherscan.io/address/${tokenAddresses.farmTwoAddress}`}
+              value={displayFromWei(totalStakedEthDpiLpBalance)}
+            />
+          </Box>
+        </Split>
+        <Spacer />
+        <Split>
+          <Box row>
+            <FancyValue
+              iconComponent={ethMviTokenIcon}
+              label='Uniswap ETH/MVI LP balance'
+              link={`https://etherscan.io/address/${tokenAddresses.uniswapEthMviLpTokenAddress}`}
+              value={displayFromWei(uniswapEthMviLpBalance)}
+            />
+          </Box>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'Staked Uniswap ETH/MVI LP Icon',
+                src: 'https://set-core.s3.amazonaws.com/img/coin-icons/uni_lp.svg',
+              }}
+              label='Staked Uniswap ETH/MVI LP'
+              link={`https://etherscan.io/address/${tokenAddresses.mviStakingRewardsAddress}`}
+              value={displayFromWei(stakedUniswapEthMviLpBalance)}
+            />
+          </Box>
+        </Split>
+      </>
+    )
+  }
+
+  const polygonModalContent = () => {
+    return (
+      <>
+        <Split>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'Defi Pulse Icon',
+                src: dpiTokenImage,
+              }}
+              label='DPI balance'
+              link={`https://polygonscan.com/address/${tokenAddresses.dpiTokenAddress}`}
+              value={displayFromWei(dpiBalancePolygon)}
+            />
+          </Box>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'MVI Icon',
+                src: 'https://set-core.s3.amazonaws.com/img/portfolios/mvi.svg',
+              }}
+              label='Metaverse Index Balance'
+              link={`https://polygonscan.com/address/${tokenAddresses.mviTokenAddress}`}
+              value={displayFromWei(mviBalancePolygon)}
+            />
+          </Box>
+        </Split>
+        <Spacer />
+        <Separator />
+        <Spacer />
+        <Split>
+          <Box row>
+            <FancyValue
+              icon={{
+                alt: 'ETH2x-FLI-P Icon',
+                src: 'https://set-core.s3.amazonaws.com/img/portfolios/eth2x_fli.svg',
+              }}
+              label='ETH 2x FLI-P Balance'
+              link={`https://polygonscan.com/address/${tokenAddresses.eth2xfliTokenAddress}`}
+              value={displayFromWei(ethflipBalance)}
+            />
+          </Box>
+        </Split>
+      </>
+    )
+  }
+
   return (
     <Modal isOpen={isOpen} onDismiss={onDismiss}>
       <StyledModalBody>
         <ModalTitle text='My Wallet' />
```

Comment:
Can we label the wallet in the UI?
`My Ethereum Wallet` and `My Polygon Wallet`